src/main/java/org/myrobotlab/service/WebkitSpeechRecognition.java

```diff
@@ -198,22 +199,26 @@ public void startListening() {
   @Override
   public void stopListening() {
     log.debug("Stop listening event seen.");
+    commError = false;

```

Comment:
this variable should be defined locally to this method if it's not used outside of this method.